pages/Profile/Profile.module.css

```diff
@@ -1,6 +1,61 @@
-.main {
-    padding: 35px;
-    font-size: x-large;
+.all {
     display: flex;
+    flex-direction: column;
     justify-content: center;
+    margin: 92px;
+    width:100vh;
+    height: 100%;
+    min-height: 100%;
 }
+
+.h2 {
+    margin-left: 5%;
+    margin-right: 5%;
+}
+
+.namebar {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 10px;
+}
+
+.boxes {
+    width: 100vh;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    margin-top: 10px;
+    gap: 4%;
+}
+
+.box {
+    padding: 5%;
+    align-content: space-between;
+    align-items: center;
+    border: 1px ;
+    border-radius: 25px;
+    border-style: outset;
+    width: 100%;
+    background-color: white;
+}
+
+.profile {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-content: center;
+    align-items: center;
+}
+
+.button {
+    background-color: #253C85;
```

Comment:
With hard-coded hex colors, if they don't exist within the global.css file, just add them in there and use the color in your css as `var(--your-color)`. This allows you to reuse the colors in other css files and you'll only have to change the hex code in one file!